.devcontainer/Dockerfile

```diff
@@ -161,7 +167,7 @@ RUN wget --progress=dot:giga https://ftp.gnu.org/gnu/nettle/nettle-2.5.tar.gz &&
     rm -rf gnutls*
 
 ##### Useful for logfile modification e.g. pruning all /magma/... prefix from GCC warning logs
-RUN GOBIN="/usr/bin/" go get github.com/ezekg/xo
+RUN GOBIN="/usr/bin/" go install github.com/ezekg/xo@0f7f076932dd
```

Comment:
do we have any process of bumping such pinned packages?

Reply:
Not as far as I know. But I wondered that as well. Let's think of something then?!
Background: I pinned it here since Hunter advised so for a different module on the go upgrade PR. But before (with go get) there was no pinning in place either and no such process necessary.